src/content/en/fundamentals/discovery-and-monetization/payment-request/index.md

```diff
@@ -40,7 +40,8 @@ like to talk about a more comprehensive solution.
 
 ## Introducing the Payment Request API {: #introducing }
 
-The Payment Request API is a system that is meant to *eliminate checkout forms*.
+The Payment Request API is a [W3C standard](https://www.w3.org/TR/payment-request/)
+that is meant to *eliminate checkout forms*.
```

Comment:
Maybe add?

'This API is designed to be vendor-agnostic meaning it does not require use of a particular payment system such as Android Pay or Apple Pay.'